lib/src/database/hive_database.dart

```diff
@@ -1403,6 +1403,22 @@ class FamedlySdkHiveDatabase extends DatabaseApi with ZoneTransactionMixin {
 
   @override
   Future<void> delete() => Hive.deleteFromDisk();
+
+  @override
+  Future<void> markUserProfileAsOutdated(userId) async {
+    return;
```

Comment:
probably should be throwing Unimplemented errors here.

Reply:
No, then the database would break on every sync. Just returning nothing leads to having no cache functionality which still makes the database able to migrate to something different